tests/terraform/static/collector_fields.output.yaml

```diff
@@ -163,46 +227,116 @@ data:
+        for FIELD in "${FIELDS[@]}" ; do
+            FIELD_ID=$( echo "${FIELDS_RESPONSE}" | jq -r "select(.fieldName == \"${FIELD}\") | .fieldId" )
+            # Don't try to import non existing fields
+            if [[ -z "${FIELD_ID}" ]]; then
+                continue
+            fi
+  
+            terraform import \
+                -var="create_fields=${CREATE_FIELDS}" \
+                sumologic_field."${FIELD}" "${FIELD_ID}"
+        done
+    fi
+  
+    # Sumo Logic Collector and HTTP sources
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_collector.collector "$COLLECTOR_NAME"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.default_events_source "$COLLECTOR_NAME/events"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.default_logs_source "$COLLECTOR_NAME/logs"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.apiserver_metrics_source "$COLLECTOR_NAME/apiserver-metrics"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.control_plane_metrics_source "$COLLECTOR_NAME/control-plane-metrics"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.controller_metrics_source "$COLLECTOR_NAME/kube-controller-manager-metrics"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.default_metrics_source "$COLLECTOR_NAME/(default-metrics)"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.kubelet_metrics_source "$COLLECTOR_NAME/kubelet-metrics"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.node_metrics_source "$COLLECTOR_NAME/node-exporter-metrics"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.scheduler_metrics_source "$COLLECTOR_NAME/kube-scheduler-metrics"
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        sumologic_http_source.state_metrics_source "$COLLECTOR_NAME/kube-state-metrics"
   
     # Kubernetes Secret
-    terraform import kubernetes_secret.sumologic_collection_secret sumologic/sumologic
+    terraform import \
+        -var="create_fields=${CREATE_FIELDS}" \
+        kubernetes_secret.sumologic_collection_secret sumologic/sumologic
   
-    terraform apply -auto-approve
+    # Apply planned changes
+    terraform apply -auto-approve \
+        -var="create_fields=${CREATE_FIELDS}"
   
     # Cleanup env variables
     export SUMOLOGIC_BASE_URL=
     export SUMOLOGIC_ACCESSKEY=
     export SUMOLOGIC_ACCESSID=
   
     bash /etc/terraform/custom.sh
-  variables.tf: |-
+  variables.tf: |
```

Comment:
As far as I understand this I needed to change this because I've added a new line at the end of `variables.tf` file.

Since all other section (apart from `providers.tf` which I guess is a mistake?) have a new line at EOF I'm changing this as well to use `  variables.tf: |` instead of `  variables.tf: |-`

cc: @sumo-drosiek @perk-sumo

Reply:
Thats correct, thats fine